Neos.Eel/Resources/Private/Grammar/AbstractParser.peg.inc

```diff
@@ -12,7 +12,7 @@ namespace Neos\Eel;
  * source code.
  */
 
-require_once __DIR__ . '/../../../Resources/Private/PHP/php-peg/Parser.php';
+require_once __DIR__ . '/../Resources/Private/PHP/php-peg/Parser.php';
```

Comment:
Thank you! :)

And thanks for pointing this out. When I generated `AbstractParser.php` from `AbstractParser.peg.inc` for the first time, i.e. with `/../../../Resources ...`, the Unit-Tests didn't run at all due to the wrong path. So I changed it to `/../Resources/ ...`, since the resulting `AbstractParser.php` is placed in `Classes/`. That's why I'm pretty sure this is correct. And not only the Unit-Tests have worked as expected by then, but also the respective line in `AbstractParser.php` hasn't changed in the diff.